LSP.py

```diff
@@ -0,0 +1,14 @@
+class Animal:
+    def legs(self):
+        pass
+
+class Lien(Animal):
+    def legs(self):
+        return 4
+
+class Monkey(Animal):
+    def legs(self):
+        return 2
+
+def GetLegs (animal: Animal):
+    car.legs()

```

Comment:
where did you copy/paste this from? :)

Reply:
Ohhh, sorry.
I have made a mistake.
At first I made example with cars but then my friend told me that my example is wrong and advised me to do example with animal legs. And I forgot to change car to animal.